clickhouse-native-jdbc/src/main/java/com/github/housepower/jdbc/serializer/BinaryDeserializer.java

```diff
@@ -66,6 +66,12 @@ public boolean readBoolean() throws IOException {
         return (container.get().readBinary() != 0);
     }
 
+    public byte[] readBytesBinary() throws IOException {
+        byte[] data = new byte[(int) readVarInt()];
+        container.get().readBinary(data);
+        return data;
+    }
+
     public String readStringBinary() throws IOException {
```

Comment:
is this method not obsolete now?

Reply:
i am not really happy with this overload, as one returns a byte[] the other a string and you can't be sure which one is called.

Reply:
No, I think we should use UTF-8 for the column name and the column type by default.
Do you agree with that?

```
String name = deserializer.readStringBinary();
```

Reply:
Ah it is for the column name. I agree with that, although most people will use US-ASCII only :-)

It maybe a good idea to rename the method returning the byte[].

Reply:
ah sorry, you renamed it. My fault!

Reply:
>  i am not really happy with this overload, as one returns a byte[] the other a string and you can't be sure which one is called.

I just checked, ClickHouse is using `US-ASCII` for column name and type, so it's better to rename it to  `readUTF8StringBinary` as you suggested.